### PR TITLE
Add support for Self-Hosted Webhook URL Usage and added project_id into the webhook payload

### PR DESCRIPTION
**Pull request by JakobStadlhuber**
This commit introduces the capability of using a Self-Hosted Webhook URL. The application now checks for a self-hosted URL before querying the database for the webhook settings. If a Self-Hosted Webhook URL is set in the environment variables, it will be used directly, diminishing unnecessary database queries.

### CODE REVIEW

**Comment by JakobStadlhuber** [comment]
#214

**Review by rafaelsideguide — APPROVED** [review]
lgtm

File: apps/api/src/services/webhook.ts [hunk 1]
```diff
@@ -28,13 +35,14 @@ export const callWebhook = async (teamId: string, data: any) => {
...
 
-  await fetch(webhooksData[0].url, {
+  await fetch(webhookUrl, {
     method: 'POST',
     headers: {
       'Content-Type': 'application/json',
     },
     body: JSON.stringify({
       success: data.success,
+      project_id: data.project_id,
```
Comment (nickscamara) [inline apps/api/src/services/webhook.ts:45]: What is this project_id again?

Comment (nickscamara) [reply apps/api/src/services/webhook.ts:45]: Besides this question, everything else looks great

Comment (nickscamara) [reply apps/api/src/services/webhook.ts:45]: Nevermind

**Comment by nickscamara** [comment]
lgtm!

**Pull request merged**